source/system/string/string_literal.c

```diff
@@ -1,31 +1,58 @@
 #include "string_literal.h"
 #include <assert.h>
 #include <ooc/system/String-fwd.h>
+#include <ooc/system/Mutex.h>
```

Comment:
This is probably better actually, then we get the correct `define`s for `pthread.h` and `windows.h` (as they are written in `system/external`).